slycot/src/AB13AD.f

```diff
@@ -260,8 +261,10 @@ DOUBLE PRECISION FUNCTION AB13AD( DICO, EQUIL, N, M, P, ALPHA, A,
 C
       IF( INFO.NE.0 ) THEN
 C
-C        Error return.
+C     Error return.
+C     
 C
+         AB13AD = ZERO
```

Comment:
I didn't do that in a separate branch though. Would be some work to split it off.